diff --git a/node_modules/orbit-db-test-utils/src/config/browser.js b/node_modules/orbit-db-test-utils/src/config/browser.js
index f8274fa..84339eb 100644
--- a/node_modules/orbit-db-test-utils/src/config/browser.js
+++ b/node_modules/orbit-db-test-utils/src/config/browser.js
@@ -1,7 +1,9 @@
+const path = require('path');
+
 module.exports = {
   timeout: 30000,
-  identityKeyFixtures: './test/fixtures/keys/identity-keys',
-  signingKeyFixtures: './test/fixtures/keys/signing-keys',
+  identityKeyFixtures: (dir) => path.resolve(dir, 'fixtures/keys/identity-keys'),
+  signingKeyFixtures: (dir) => path.resolve(dir, 'fixtures/keys/signing-keys'),
   identityKeysPath: './orbitdb/identity/identitykeys',
   signingKeysPath: './orbitdb/identity/signingkeys',
   defaultIpfsConfig: {
diff --git a/node_modules/orbit-db-test-utils/src/config/node.js b/node_modules/orbit-db-test-utils/src/config/node.js
index 25c52a1..78fc487 100644
--- a/node_modules/orbit-db-test-utils/src/config/node.js
+++ b/node_modules/orbit-db-test-utils/src/config/node.js
@@ -1,7 +1,9 @@
+const path = require('path');
+
 module.exports = {
   timeout: 30000,
-  identityKeyFixtures: './test/fixtures/keys/identity-keys',
-  signingKeyFixtures: './test/fixtures/keys/signing-keys',
+  identityKeyFixtures: (dir) => path.resolve(dir, 'fixtures/keys/identity-keys'),
+  signingKeyFixtures: (dir) => path.resolve(dir, 'fixtures/keys/signing-keys'),
   identityKeysPath: './orbitdb/identity/identitykeys',
   signingKeysPath: './orbitdb/identity/signingkeys',
   defaultIpfsConfig: {
diff --git a/node_modules/orbit-db-test-utils/src/mem-store.js b/node_modules/orbit-db-test-utils/src/mem-store.js
index 76660c7..120a21a 100644
--- a/node_modules/orbit-db-test-utils/src/mem-store.js
+++ b/node_modules/orbit-db-test-utils/src/mem-store.js
@@ -22,11 +22,11 @@ const cidifyString = (str) => {
 
 /* Memory store using an LRU cache */
 class MemStore {
-  constructor () {
+  constructor() {
     this._store = new Map()
   }
 
-  async put (value) {
+  async put(value) {
     const multihash = await sha256.digest(json.encode(value))
     const cid = CID.create(1, DAG_CBOR_CODE, multihash)
     const key = cid.toString(defaultBase)
@@ -36,7 +36,7 @@ class MemStore {
     return cid
   }
 
-  async get (cid) {
+  async get(cid) {
     const data = this._store.get(cid.toString(defaultBase))
 
     // TODO: Change this to refs
